crates/biome_cli/src/execute/process_file/search.rs

```diff
@@ -21,16 +21,33 @@ pub(crate) fn search_with_guard<'ctx>(
 ) -> FileResult {
     tracing::info_span!("Processes searching", path =? workspace_file.path.display()).in_scope(
         move || {
-            let _result = workspace_file
+            let result = workspace_file
                 .guard()
                 .search_pattern(pattern)
                 .with_file_path_and_code(
                     workspace_file.path.display().to_string(),
                     category!("search"),
                 )?;
 
+            let input = workspace_file.input()?;
+            let file_name = workspace_file.path.display().to_string();
+
             // FIXME: We need to report some real results here...
```

Comment:
I guess so, since the formatting is ready for the engine that provides real results is :)

Reply:
Actually no, I have left it there by mistake, I will remove it 👍

Reply:
> I guess so, since the formatting is ready for the engine that provides real results is :)

Sorry for my last out-of-sync reply: I actually went ahead and removed this one, in favor of keeping the other `TODO` inside the function which is actually supposed to return the matches